service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/OrdersSoughtChecker.java

```diff
@@ -59,6 +67,10 @@ public boolean isCompleted(CaseData caseData) {
             }
         }
 
+        if (orders.isSecureAccommodationOrder() && isSecureAccomodationOrderNotCompleted(orders)) {
```

Comment:
Bit of a weird one, as I can't add a comment to the right line 😆

was just checking the orders.isSecureAccomodationOrder() function, I think it needs a null/empty check in it (similar to the containsEPO/isDischargeOfCare) otherwise the validator might break before we fill in that section

Reply:
Oh right! Added null checking